### DISCUSSION TITLE
Will FreetzNG provide better SQM/QoS for Gaming than regular FritzOS?

### DISCUSSION THREAD

**Discussion by yCr-shiddy** [Freetz-NG/freetz-ng#789] [Allgemein]
Hey, as title shows - I wonder if FreetzNG are more helpful for onlinegaming and/or Bufferbloat. I managed to get it nearly to 0ms due OpenWRT Router and proper SQM/QoS like Cake but asked myself if Freetz are helpful for people that not have an additional Router. 
Also will VoIP telephone works as it should on FritzOS?
 Thank you :)

**Comment by fda77** [comment]
Avm uses closed-source daemons for routing. Most "normal" linux tools dont work
